test/evergreen.yml

```diff
@@ -732,11 +732,9 @@ functions:
           fi
           ${virtualenv_binary} -p ${python_binary} venv
           source venv/bin/activate
-          ${pip3_binary} install psutil pygit2==1.10.1
-          JSON_TASK_INFO='{ "evergreen_task_info": { "is_patch": "'${is_patch}'", "task_id": "'${task_id}'", "distro_id": "'${distro_id}'", "execution": "'${execution}'", "task_name": "'${task_name}'", "version_id": "'${version_id}'", "branch_name": "'${branch_name}'" } }'
-          echo "JSON_TASK_INFO: $JSON_TASK_INFO"
-          ${test_env_vars|} ${python_binary} ../../../bench/perf_run_py/perf_run.py --${test_type|wtperf} -e ${exec_path|./wtperf} -t ${perf-test-path|../../../bench/wtperf/runners}/${perf-test-name} -ho WT_TEST -m ${maxruns} -g "../.." -v -i "$JSON_TASK_INFO" -b -o test_stats/evergreen_out_${perf-test-name}.json ${wtarg}
-          ${test_env_vars|} ${python_binary} ../../../bench/perf_run_py/perf_run.py --${test_type|wtperf} -e ${exec_path|./wtperf} -t ${perf-test-path|../../../bench/wtperf/runners}/${perf-test-name} -ho WT_TEST -m ${maxruns} -g "../.." -v -i "$JSON_TASK_INFO" -re -o test_stats/atlas_out_${perf-test-name}.json ${wtarg}
+          ${pip3_binary} install psutil
```

Comment:
Since we're modifying this line can we lock the version of psutil? 
Looking at [one of our recent commits](https://evergreen.mongodb.com/task/wiredtiger_ubuntu2004_perf_tests_perf_test_evict_btree_1_4a23a5846b31a014e2ec5c169bac856c9a942d1f_23_03_08_09_01_09/0) we're currently using version 5.9.4.

Reply:
Sure, updated in [835bcbe](https://github.com/wiredtiger/wiredtiger/pull/8904/commits/835bcbe11d2ec56b12b7d16152f07619963578ae)